README.md

```diff
@@ -104,11 +104,54 @@ If you are using the types in a `commonjs` module, like in a Node app, you just
 ## Trouble shooting and known issues.
 
 ### Web3 and Angular
+
+### New solution
+
+If you are using Angular version >11 and run into an issue building, the old solution below will not work. This is because polyfills are not included in the newest version of Angular.
+
+- Install the required dependencies within your angular project:
+
+```bash
+npm install --save-dev crypto-browserify stream-browserify assert stream-http https-browserify os-browserify
+```
+
+- Within `tsconfig.json` add the following `paths` in `compilerOptions` so Webpack can get the correct dependencies
+
+```typescript
+{
+    "compilerOptions": {
+        "paths" : {
+        "crypto": ["./node_modules/crypto-browserify"],
+        "stream": ["./node_modules/stream-browserify"],
+        "assert": ["./node_modules/assert"],
+        "http": ["./node_modules/stream-http"],
+        "https": ["./node_modules/https-browserify"],
+        "os": ["./node_modules/os-browserify"],
+    }
+}
+```
+
+- Add the following lines to `polyfills.ts` file:
+
+```typescript
+import { Buffer } from 'buffer';
+
+(window as any).global = window;
+global.Buffer = Buffer;
+global.process = {
+    env: { DEBUG: undefined },
+    version: '',
+    nextTick: require('next-tick')
+} as any;
```

Comment:
Should not following webpack config works instead of pathing the global object. 

```
plugins: [
      new webpack.ProvidePlugin({
             process: 'process/browser',
      }),
  ],
```

Reply:
Angular has a custom `webpack.config.js` and is hidden from the user, so editing the webpack config requires installing dependancies